dipy/workflows/tracking.py

```diff
@@ -115,17 +113,19 @@ def run(self, pam_files, stopping_files, seeding_files,
            Path to the peaks and metrics files. This path may contain
             wildcards to use multiple masks at once.
         stopping_files : string
-            Path of FA or other images used for stopping criteria for tracking.
+            Path to images (e.g. FA) used for stopping criteria for tracking.
         seeding_files : string
             A binary image showing where we need to seed for tracking.
         stopping_thr : float, optional
             Threshold applied to stopping volume's data to identify where
-             tracking has to stop (default 0.25).
+            tracking has to stop (default 0.25).
```

Comment:
This space is on purpose because of the space stripping when you display the help

Reply:
you can forget this comment